### DISCUSSION TITLE
How to get more info why did photoprism flagged my videos as hidden or missing?

### DISCUSSION THREAD

**Discussion by gabrielnemeth** [photoprism/photoprism#2044] [Q&A] [answered]
Hey,

I'm using the latest photoprism on docker. A week ago did a library index re-scan and noticed an error afterwards in the library tab: `The index currently contains 127 hidden files. Their format may not be supported, they haven't been converted to JPEG yet or there are duplicates.` Checked the hidden tab and the videos aren't duplicated nor corrupted since I can play them through photoprism if I click on them. So I went to check the logs, but there was no meaningful log about why does this happened: `index: flagged 8 photos as hidden or missing [72.444391ms]` was the only related log.

What I did try already:
- installed older versions of photoprism
- did a complete re-scan couple times
- did a database reset through `photoprism reset`

**Comment by lastzero** [comment, marked as answer]
https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Reply by gabrielnemeth** [reply]
Dam it, I had the `Convert to JPEG` turned off. Thank you!
